tests/models/convbert/test_modeling_convbert.py

```diff
@@ -437,6 +437,17 @@ def test_torchscript_device_change(self):
                 loaded = torch.jit.load(os.path.join(tmp, "traced_model.pt"), map_location=torch_device)
                 loaded(inputs_dict["input_ids"].to(torch_device), inputs_dict["attention_mask"].to(torch_device))
 
+    def test_model_for_input_embeds(self):
+        batch_size = 2
+        seq_length = 10
+        inputs_embeds = torch.rand([batch_size, seq_length, 768])
+        config = self.model_tester.get_config()
+        model = ConvBertModel(config=config)
+        model.to(torch_device)
+        model.eval()
+        result = model(inputs_embeds=inputs_embeds)
+        self.assertEqual(result.last_hidden_state.shape, (batch_size, seq_length, config.hidden_size))
```

Comment:
Nice thanks for thinking of adding a tests. Let's add slow to make sure this is ran on a GPU

Reply:
Done

Reply:
All tests are run on GPU, you don't need to put `@slow` for that @ArthurZucker ;-)

Reply:
Oups, I though I had removed that comment! 😓 The device is automatically deduced even if there are no GPU